<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="0" failures="0" file="Approve_tests.py" name="Approve_tests.ApproveTest-20191030160642" skipped="0" tests="2" time="0.005" timestamp="2019-10-30T16:06:42">
	<testcase classname="Approve_tests.ApproveTest" file="src/unittest/python/Approve_tests.py" line="12" name="testCanApprove" time="0.003" timestamp="2019-10-30T16:06:42">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="Approve_tests.ApproveTest" file="src/unittest/python/Approve_tests.py" line="18" name="testDescription" time="0.002" timestamp="2019-10-30T16:06:42">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
</testsuite>
